app/components/search/SearchMap/SearchMap.tsx

```diff
@@ -17,34 +17,32 @@ import config from "../../../config";
 
 export const SearchMap = ({
   hits,
-  mapObject,
-  setMapObject,
-  setAroundLatLng,
   mobileMapIsCollapsed,
 }: {
   hits: TransformedSearchHit[];
-  mapObject: google.maps.Map | null;
-  setMapObject: (map: any) => void;
-  setAroundLatLng: (latLng: { lat: number; lng: number }) => void;
   mobileMapIsCollapsed: boolean;
 }) => {
-  const { userLocation } = useAppContext();
-  if (userLocation === null) {
-    return (
-      <div className="mapLoaderContainer">
-        <Loader />
-      </div>
-    );
-  }
+  const [googleMapObject, setMapObject] = useState<google.maps.Map | null>(
+    null
+  );
+  const { userLocation, aroundLatLng } = useAppContext();
+  const { setAroundLatLng } = useAppContextUpdater();
 
   function handleSearchThisAreaClick() {
-    const center = mapObject?.getCenter() || null;
+    const center = googleMapObject?.getCenter();
     if (center?.lat() && center?.lng()) {
-      setAroundLatLng({ lat: center.lat(), lng: center.lng() });
+      setAroundLatLng(`${center.lat()}, ${center.lng()}`);
```

Comment:
Note: The value type has now changed from an object to a string to standardize it's representation anywhere in the application. Depending on future needs, this might actually be a candidate for a Value Object or a plain object again.